docs/source/development/linting.md

```diff
@@ -40,20 +41,28 @@ pip install black
 pip install flake8
 pip install isort
 ```
+#### Configure `flake8`
+
+Store your `flake8` configuration in a file named `setup.cfg` within your project root. The Kedro starters use the [following configuration](https://github.com/kedro-org/kedro-starters/blob/main/pandas-iris/%7B%7B%20cookiecutter.repo_name%20%7D%7D/setup.cfg):
```

Comment:
Notably, `flake8` is among the few tools (maybe the only one) that can't be configured in `pyproject.toml`